unlocks.csv

```diff
@@ -389,13 +389,13 @@ Analysis|LaserCutter|0,0,LaserCutter
 Goal|OnCraftBattery|0,1,Battery
 Goal|OnCraftComputerChip|0,1,ComputerChip|CopperWire
 Goal|OnCraftTank|0,1,Tank
-Goal|OnCraftDoubleTank|0,1,DoubleTank|Glass|Tank
-Goal|UnlockHatchingEnzymes|0,1,HatchingEnzymes
-Goal|UnlockRadiationSuit|0,1,
-Goal|UnlockRadiationSuit|1,1,
-Goal|UnlockRadiationSuit|2,1,
-Goal|Aurora_RingRoom_Terminal2|0,1,Welder|LaserCutter|Battery
-Goal|Aurora_RingRoom_Terminal3|0,1,Welder|LaserCutter|Battery
+Goal|OnCraftDoubleTank|0,0,DoubleTank|Glass|Tank
+Goal|UnlockHatchingEnzymes|0,0,
```

Comment:
Safety mechanism in case hatching enzymes is in a weird place. Also removed the 'softlock' protection - this is the flag for unlocking the recipe normally, not requiring the item, so does not matter.